concert/experiments/base.py

```diff
@@ -70,7 +70,7 @@ async def _run(self):
         await asyncio.gather(*coros, return_exceptions=False)
 
     @background
-    @check(source=['standby', 'error'], target='standby')
+    @check(source=['standby', 'error'], target=['standby', 'cancelled'])
```

Comment:
"cancelled" should also be in source, otherwise we can  not re-run an experiment or acquisition, if it was cancelled before.
In https://github.com/ufo-kit/concert/pull/504/commits/6c55a59021660c269966b19ca43af1e2c1b6426d I added them (also to all the other places in the experiments).
About the target I'm not 100% sure if we add it there or raise the Exception, that the final state is not correct.

Reply:
Right, actually it should raise the exception. I just swallow it now but when an experiment is used in a director it must notify it. I falsely assumed the experiment is at the top of the food chain, so I'd change that actually to raise the `CancelledError`, ok?